pkg/virt-controller/watch/migration.go

```diff
@@ -48,6 +48,8 @@ import (
 	"kubevirt.io/kubevirt/pkg/virt-controller/services"
 )
 
+const failedToProcessDeleteNotificationErrMsg = "Failed to process delete notification"
```

Comment:
This seems to fit under the `deletePod` method where it is used and not at this global level.

Reply:
This error message is being used in two functions: `deletePod` and `deleteVMI`. Therefore I think it's okay to keep it at a global scope.

Reply:
Right, I missed the other usage. Sorry.